app/models/validations/financial_validations.rb

```diff
@@ -98,11 +98,11 @@ def validate_rent_amount(record)
   end
 
   def validate_rent_period(record)
-    if record.owning_organisation.present? && record.owning_organisation.rent_periods.present? &&
```

Comment:
Is the `managing_organisation` always set?

Reply:
Not when it's created by the support user